client/components/cards.jsx

```diff
@@ -0,0 +1,264 @@
+import React from 'react';
+import AppContext from '../lib/app-context';
+
+const buttons = [
+  { name: 'Select Card', action: 'select-card' },
+  { name: 'Ready', action: 'ready' }
+];
+
+const styles = {
+  button: {
+    width: '130px',
+    height: '40px',
+    borderRadius: '7px',
+    backgroundColor: '#dbdbdb',
+    border: 'none',
+    margin: '1%'
+  }
+};
+
+export default class Cards extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      cards: null,
+      images: null,
+      cardShown: 0,
+      isReadyDisabled: false,
+      message: '',
+      messageClass: ''
+    };
+
+    this.handleDotClick = this.handleDotClick.bind(this);
+    this.handleNext = this.handleNext.bind(this);
+    this.handlePrevious = this.handlePrevious.bind(this);
+  }
+
+  componentDidMount() {
+    this.getCards();
+    this.getImages();
```

Comment:
Generally it is best to get all of your data from a single request. No need to change this but something to keep in mind moving forward.